coins.json

```diff
@@ -507,7 +507,7 @@
         "name": "Steem",
         "symbol": "STEEM",
         "decimals": 3,
-        "blockchain": "Ethereum",
+        "blockchain": "EOS",
```

Comment:
@hewigovens steem would be a separate blockchain type, their protocol not compatible